mirdata/beatport_key.py

```diff
@@ -90,8 +90,8 @@ def __init__(self, track_id, data_home=None):
 
     @utils.cached_property
     def key(self):
-        """String: key annotation"""
-        return load_key(self.keys_path)
+        """List of String: list of possible key annotations"""
+        return load_key(self.keys_path).split(' | ')
```

Comment:
can the `load_key` method do the `.split` there instead of in this property?

Reply:
yes of course!

Reply:
@rabitt I know that we speak about if to use or not use the keydata structure. I defend that it is overengineering. However now that we are returning a list I think that is better. And I am going to set endtime = 120s and first_time = 0s in every key. I think that is more clear that. the two keys can be possible.

Reply:
done here. ba05b89

Reply:
What about returning KeyData, as you do in the commit, but just not passing it in the `jams_converter` call?

Also, my point about doing the `.split` within the `load_key` method was so that someone using off-disk data can do
`load_key(custom_keys_path)` and get the same result as if they did `Track.key`

Reply:
I moved the .split to load here: 5688a17  but I reverted it !
I have added it other time! 008739c

Reply:
>> What about returning KeyData, as you do in the commit, but just not passing it in the jams_converter call?

I think that there is a huge relationship between giant steps (difficult keys) and Beatport keys (easier keys). The two datasets are to detect keys in EDM mixes... And they have to have consistency between them because people who use one should use the other one. If we can not introduce it in the key space... i think that is better if we keep the format as now (the same format that giant steps)

Reply:
What do you think @rabitt ?